### PR TITLE
Updated Podfiles in the guides to the new syntax

### PR DESCRIPTION
**Pull request by orta**
Waiting on a release

### CODE REVIEW

File: source/using/the-podfile.html.md [hunk 1]
```diff
@@ -3,56 +3,70 @@ title: The Podfile
...
     pod 'OCMock', '~> 2.0.1'
+  end
 end
 
 post_install do |installer|
-    installer.pods_project.targets.each do |target|
-        puts target.name
-    end
+  installer.pods_project.targets.each do |target|
+    puts "#{target.name}"
```
Comment (segiddins) [inline source/using/the-podfile.html.md]: Why this change?

File: source/using/the-podfile.html.md [hunk 2]
```diff
@@ -104,13 +109,13 @@ Pods often depend on other pods. Conflicts arise when multiple pods depend on di
 > The conflict error looks like this:
 
 ```shell
-[!] Podfile tries to activate `GoogleAnalytics-iOS-SDK (= 2.0beta4)', but already activated version `3.0' by ARAnalytics/GoogleAnalytics (1.6).
```
Comment (segiddins) [inline source/using/the-podfile.html.md]: Isn't this whole section outdated now we use Molinillo?

File: source/using/using-cocoapods.html.md [hunk 3]
```diff
@@ -47,7 +52,7 @@ pod 'ObjectiveSugar'
 
 ### Integration with an existing workspace
 
-Integrating CocoaPods with an existing workspace requires one extra line in your Podfile. Simply specify the `.xcworkspace` root filename like so:
+Integrating CocoaPods with an existing workspace requires one extra line in your Podfile. Simply specify the `.xcworkspace` filename in a target block like so:
```
Comment (segiddins) [inline source/using/using-cocoapods.html.md]: The workspace actually doesn't go in a target block

File: source/using/the-podfile.html.md (hunk 2 above)
Comment (orta) [inline source/using/the-podfile.html.md]: no idea, maybe? :D

File: source/using/the-podfile.html.md (hunk 1 above)
Comment (orta) [inline source/using/the-podfile.html.md]: that's what it is in CocoaPods/Core - I C&P'd it in

**Comment by orta** [comment]
Addressed feedback :+1:

File: source/using/the-podfile.html.md (hunk 1 above)
Comment (orta) [inline source/using/the-podfile.html.md]: figure its to deal with nils better? dunno

File: source/using/using-cocoapods.html.md [hunk 4]
```diff
@@ -29,17 +31,20 @@ To create a new project with CocoaPods, follow these simple steps:
...
+* Create a Podfile. This can be done by running `$ pod init`.
 * Open your Podfile. The first line should specify the platform and version supported.
 
 ```ruby
-platform :ios, '8.0'
+platform :ios, '9.0'
 ````
 
-* Add a CocoaPod by specifying `pod '$PODNAME'` on a single line
+* In order to use CocoaPods you need to define the Xcode target to link them to. So for example if you are writing an iOS app, it would be the name of your app. Create a target section by writing `target '$TARGET_NAME' do` and an `end` a few lines after.
```
Comment (kylef) [inline source/using/using-cocoapods.html.md]: Since the above instructions we're to use `pod init`, wouldn't the Podfile already have the target?

File: source/using/the-podfile.html.md (hunk 1 above)
Comment (segiddins) [inline source/using/the-podfile.html.md]: nope, this is totally unnecessary -- `puts nil` works just fine, and all targets will have a name anyways

**Comment by segiddins** [comment]
please rebase to get rid of the merge commit, otherwise I think this is good to merge?
